code/modules/antagonists/wizard/wizard.dm

```diff
@@ -152,12 +153,12 @@
 
 /datum/antagonist/wizard/apply_innate_effects(mob/living/mob_override)
 	var/mob/living/M = mob_override || owner.current
-	update_wiz_icons_added(M, wiz_team ? TRUE : FALSE) //Don't bother showing the icon if you're solo wizard
```

Comment:
Not keeping this ?

Reply:
As far as I can tell, the extra check in the original proc doesn't do anything. Original proc:
```
/datum/antagonist/wizard/proc/update_wiz_icons_added(mob/living/wiz,join = TRUE)
	var/datum/atom_hud/antag/wizhud = GLOB.huds[ANTAG_HUD_WIZ]
	wizhud.join_hud(wiz)
	set_antag_hud(wiz, hud_version)
```
So we have the `join` variable but there's never any checks made with it. I'm still getting the wizhud as a solo wizard on the master branch.

Reply:
All good then.